CHANGELOG.md

```diff
@@ -1,16 +1,42 @@
 Changelog
 =========
 
-## 0.6.0 SNAPSHOT (current master)
+## 0.6.0-RC1
```

Comment:
General question: do we keep the RC-Changelog or do we rename `0.6.0-RC1` later to `0.6.0` (and eventually add further entries).

Reply:
I would rename it later to `0.6.0`